azure/requests.yaml

```diff
@@ -1,20 +1,31 @@
 releases:
     - name: ">= 15.0.0"
       requests:
+        - name: app-operator
+          version: ">= 4.1.0"
+          issue: https://github.com/giantswarm/roadmap/issues/254
```

Comment:
With this we create an app-operator per workload cluster. This is new functionality so requesting for the next major release.

In the release CR releaseOperatorDeploy needs to be set to false.